Clarify sentence about "last_affected" and "fixed" in docs. (#310)
ossf/osv-schema: docs/schema.md

`limit`) is allowed in each event object. For instance,
`{"introduced": "1.0.0", "fixed": "1.0.2"}` is **invalid**.

> An events array can have entries containing either "last_affected" or "fixed" events, 

Reviewer: Yes, there's some semantic nuance here, which is a little tricky to convey, I think.

So to get all JSON-technical, the `events` array is an array of JSON (event) _objects_, and the intention of the statement under scrutiny is to refer to these event objects, as you've surmised.

https://github.com/ossf/osv-schema/blob/main/validation/schema.json#L92-L150

```suggestion
Entries in the `events` array may be "last_affected" or "fixed" events, 
```

Author: @andrewpollock I think that it's possible that someone may still misread that suggestion as disallowing this case:

```
[..., {"last_affected": "1.0.0", "fixed": "1.0.1"}, ...]
```

Reviewer: the "but not both" still remains, so that should be pretty clear?
